libs/core/langchain_core/callbacks/manager.py

```diff
@@ -1729,7 +1729,8 @@ async def on_llm_start(
                 to each prompt.
         """
 
-        tasks = []
+        inline_tasks = []
```

Comment:
Actually give me a few -- might be quicker for me to re-write

Reply:
@parambharat I updated this code to follow the structure from the comment above. The handlers should be kept together because the majority of dispatching logic should not be done at this level, but by the underlying ahandle_event